tests/test_pca.py

```diff
@@ -83,75 +81,105 @@ def wrapper(a: np.ndarray) -> DaskArray:
 }
 
 
-@pytest.fixture(
-    params=[
-        param_with(at, marks=[needs.dask_ml]) if "dask" in at.id else at
-        for at in ARRAY_TYPES_SPARSE_DASK_UNSUPPORTED
-    ]
-)
-def array_type(request: pytest.FixtureRequest):
+def conv_at(array_type):
```

Comment:
type please but also name of the function to `maybe_convert_array_to_dask` or something?